Sources/Capriccio/CapriccioArgumentsParser.swift

```diff
@@ -48,4 +49,39 @@ final class CapriccioArgumentsParser {
         
         return CapriccioArguments(source: source, destination: destination, excludedTags: excludedTags, includedTags: includedTags, generatedClassType: generatedClassType, useSingleFile: useSingleFile, disableSwiftLint: disableSwiftLint, templateFilePath: templateFile)
     }
+    
+    static func parseArguments(yaml yamlPath: String) -> CapriccioArguments {
+        let yamlFile = try? String(contentsOfFile: yamlPath)
```

Comment:
This is a great idea!
Only one thing that I'm thinking is that probably you can make `CapriccioArguments` `Decodable` and get this for free with 
```swift
let decoder = YAMLDecoder()
do {
  return try decoder.decode(CapriccioArguments.self, from: encodedYAML)
} catch {
  fatalError("Unable to load YAML file: .capriccio.yml")
}
```